docs/editor_integration.md

```diff
@@ -233,6 +233,32 @@ If you later want to update _Black_, you should do it like this:
 $ pip install -U black --no-binary regex,typed-ast
 ```
 
+## Gedit
+gedit is the default text editor of the GNOME, Unix like Operating Systems.
+Open gedit as
+```console
+$ gedit <file_name>
+```
+1. `Go to edit > preferences > plugins`
+2.  Search for `external tools` and activate it.
+3.  In `Tools menu -> Manage external tools`
+4.  Add a new tool using `+` button.
+5.  Copy the below content to the code window.
+```console
+#!/bin/bash
+Name = $GEDIT_CURRENT_DOCUMENT_NAME
+black $Name
```

Comment:
I followed your steps (which are easy to follow btw!) and everything worked except for the actual bash script :(
![Screenshot from 2021-02-03 19-06-23](https://user-images.githubusercontent.com/63936253/106825482-f644fd80-6652-11eb-9370-e18143e574ff.png)

Skipping the "saving to a variable" step and directly providing the document path to black worked tho.

Reply:
Alright, thanks for looking into this..
fixed the variable thing... find that I put a space between 'Name' and '=', in the docs while formatting.
Again added a pull request,